elasticapm/handlers/logging.py

```diff
@@ -44,22 +45,26 @@
 
 class LoggingHandler(logging.Handler):
     def __init__(self, *args, **kwargs):
-        client = kwargs.pop("client_cls", Client)
-        if len(args) == 1:
+        self.client = None
+        if "client" in kwargs:
+            self.client = kwargs.pop("client")
+        elif len(args) > 0:
             arg = args[0]
-            args = args[1:]
             if isinstance(arg, Client):
                 self.client = arg
+
+        if not self.client:
+            client_cls = kwargs.pop("client_cls", None)
+            if client_cls:
+                self.client = client_cls(*args, **kwargs)
             else:
-                raise ValueError(
-                    "The first argument to %s must be a Client instance, "
-                    "got %r instead." % (self.__class__.__name__, arg)
+                # In 6.0, this should raise a ValueError
+                warnings.warn(
+                    "LoggingHandler requires a Client instance. No Client was "
+                    "received. This will result in an error starting in v6.0",
```

Comment:
just a small nit, but I'd explicitly say "v6.0 of the agent" to make it clear that we mean the agent version, not stack version or similar